src/functions/asyncCredential/sessionService/sessionService.ts

```diff
@@ -64,6 +66,49 @@ export class SessionService implements IRecoverAuthSession {
     return successResponse(result.Items[0].sessionId.S);
   }
 
+  async createSession(
+    sessionConfig: ISessionConfig,
```

Comment:
Should the interface here be called AuthSession?

Reply:
Updated